### PR TITLE
Working

### PR DESCRIPTION
**Pull request by jtywa**
Created a Tab Panel with options to filter the list of tasks by 'All', 'Open', 'Complete', or 'Archived. Integrated MUI components for styling and functionality. Clicking a tab calls a filter function stored in App.jsx and passed down via props. Refactored the filter function for better readability.

### CODE REVIEW

File: client/src/components/TaskList.jsx [hunk 1]
```diff
@@ -1,12 +1,28 @@
-import { Grid } from '@mui/material'
+import { Grid, styled } from '@mui/material'
 import React from 'react'
 import Task from './Task'
 
+const StyledGrid = styled(Grid)`
```
Comment (kathrynherod) [inline client/src/components/TaskList.jsx:5]: mobile
image

File: client/src/components/TaskList.jsx [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+  align-items: center;
+  display: inline-grid;
+  grid-gap: 12px;
+  grid-template-columns: repeat(1, 350px);
+  justify-content: center;
+  margin-top: 32px;
+
+  ${props => props.theme.breakpoints.up("md")} {
+    grid-template-columns: repeat(2, 350px);
```
Comment (kathrynherod) [inline client/src/components/TaskList.jsx:14]: tablets+ 
image

File: client/src/components/TaskList.jsx [hunk 3]
```diff
@@ -1,12 +1,28 @@
...
+  }
+  ${props => props.theme.breakpoints.up("lg")} {
```
Comment (kathrynherod) [inline client/src/components/TaskList.jsx:16]: Desktops
image

**Review by jtywa — COMMENTED** [review]
Starting to look great! Thank you for the changes

**Review by davidjumiller — APPROVED** [review]
LGTM! :)

**Pull request merged**